server/sublist_test.go

```diff
@@ -897,6 +1062,14 @@ func Benchmark______________________SublistInsert(b *testing.B) {
 	}
 }
 
+func Benchmark_______________SublistInsertNoCache(b *testing.B) {
```

Comment:
The "no cache" affects more the `Match()`. What I mean by that is that code change was in Match(), not Insert()

Reply:
Affects both, I wrote tests before any code so was fixing all the crashes.. ;)